### DISCUSSION TITLE
Code Example for Drawer Component

### DISCUSSION THREAD

**Discussion by soniaisiganporia** [wix/react-native-ui-lib#1454] [Q&A] [answered]
I am trying to add a Left and Right drawer to a Card component. Is there a code example I can work off of? I haven't been able to find one anywhere. Thanks in advance!

For context: the example in this documentation is exactly what I am trying to implement.
https://wix.github.io/react-native-ui-lib/docs/Drawer

**Comment by awHamer** [comment]
Here you go
https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/ConversationListScreen.js
https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/DrawerScreen.tsx

**Comment by ethanshar** [comment, marked as answer]
```
<Drawer rightItems={RIGHT_ITEMS} leftItem={LEFT_ITEM}>
    <Card>
        /* Your Card content */
  </Card>
</Drawer>
```

**Comment by awHamer** [comment]
Please not that Drawer has a big issue on android
https://github.com/wix/react-native-ui-lib/issues/1028#issuecomment-947665659
